Week1/Homework/ex1.js

```diff
@@ -0,0 +1,45 @@
+const mysql = require("mysql");
+const connections ={
+    host: "localhost",
+    user: "hyfuser",
+    password: "hyfpassword"
+  }
+const db = mysql.createConnection(connections);
+  db.connect((err) => {
```

Comment:
indentation is odd here

Reply:
sorry my teacher I did not understand what "indentation is odd here" means actually, may you explain it more please